src/cosmogony.rs

```diff
@@ -35,6 +37,15 @@ impl CosmogonyStats {
             }
         });
     }
+    pub fn process_zone_types(&mut self, zones: &Vec<Zone>) {
```

Comment:
can't we do both `init` and `process_zone_types` at the same time (doing it later then)

Reply:
:+1: